components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/SelectAcrFromFunction.java

```diff
@@ -37,7 +37,7 @@ public class SelectAcrFromFunction implements SelectOneFunction {
 
     private static final Log log = LogFactory.getLog(SelectAcrFromFunction.class);
 
-    public String evaluate(JsAuthenticationContext context, Object possibleOutcomesObj) {
+    public String evaluate(JsBaseAuthenticationContext context, Object possibleOutcomesObj) {
```

Comment:
here we are changing the method signature, is it possible to overload them?

Reply:
We don't need this change. Changed with f00358d0066bbf13b65c87f4867e2ad5052761fc